dsl/com/electriccloud/commander/dsl/util/BaseObject.groovy

```diff
@@ -371,28 +417,36 @@ abstract class BaseObject extends DslDelegatingScript {
       println "  Add overwrite flag to DSLDelegate vars: " + overwriteMode.toBoolean()
       script.getDelegate().getBinding().setVariable("overwrite", overwriteMode.toBoolean())
     }
-    println "  Add binding map to DSLDelegate vars: " + bindingMap
+    //println "  Add binding map to DSLDelegate vars: " + bindingMap
     script.getDelegate().getBinding().setVariable("bindingMap", bindingMap)
     script.getDelegate().getBinding().setVariable("pluginDeployMode", pluginDeployMode)
     return script.run();
   }
 
-  def loadAcls (File aclDir, String objPath,
-                Map bindingMap) {
-   println "Entering loadAcls"
-   println "  dir  : $aclDir"
-   println "  path : $objPath"
-   println "  map  : " + bindingMap.toMapString(250)
+  def loadAcls (File aclDir, String objPath, Map bindingMap, changeList = [:]) {
+//    println "Entering loadAcls"
```

Comment:
I don't see we use changeList inside the method

Reply:
I just added it